scripts/ros_install.bash

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e  # exit on first error
+ROS_VERSION="indigo"
+ROS_BASH="/opt/ros/$ROS_VERSION/setup.bash"
+ROS_PACKAGES_URL='http://packages.ros.org/ros/ubuntu'
+APT_KEYS_URL='hkp://pool.sks-keyservers.net:80'
+APT_TARGETS="$(lsb_release -sc) main"
+SOURCES_LIST_TARGET='/etc/apt/sources.list.d/ros-latest.list'
+
+
+install()
+{
+	# update sources.list and add apt-keys
+	echo "deb $ROS_PACKAGES_URL $APT_TARGETS" > $SOURCES_LIST_TARGET
+	apt-key adv --keyserver $APT_KEYS_URL --recv-key "0xB01FA116"
+
+	# update apt and install ros
+	apt-get update
+	apt-get install -y ros-$ROS_VERSION-desktop-full
+
+	# initialize rosdep
+	rosdep init
+	rosdep update
+
+	# env setup
+	echo "source /opt/ros/$ROS_VERSION/setup.bash" >> $HOME/.bashrc
+
+	# install ros
+	apt-get install -y python-rosinstall
+
+	# install ros packages	
```

Comment:
Ah, sorry I didn't read your comment thoroughly enough.  From experience excluding the packages led to compilation failures, doing a quick google search gave me this [link](http://answers.ros.org/question/154806/indigo-on-trusty/). Plus `apt-get` actually checks to see if packages are already installed so its inclusion isn't a bad thing, I would argue that I should do a lite ROS desktop install, so that we have a good understanding of what packages does mcptam actually depend on.